pages/api/reviews/index.ts

```diff
@@ -0,0 +1,84 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import Review from './model/ReviewSchema';
+import { connectToDatabase } from '../../../Utils/mongodb';
+
+const saveReview = async (req: any, res: any) => {
+    try {
+        await connectToDatabase();
+        // TODO: Need to take revieweeId from token which user is login
+        // if not then send error response
+
+        const reviewInfo = req.body;
+
+        // check if reviewer already give review to user
+        const isReviewExistGivenByRevieweer = await Review.findOne({
+            revieweeId: reviewInfo.revieweeId,
+            reviewerId: reviewInfo.reviewerId,
+        });
+
+        if (isReviewExistGivenByRevieweer) {
+            return res.json({
+                code: 400,
+                status: 'error',
+                error: 'You already give review for this user',
+            });
+        }
+        // TODO:
+        // need to check if reviewer is current or previous colleague
+        // if yes then user can give review else can not give review
+
+        await Review.create(reviewInfo);
+
+        return res.json({ code: 201, status: 'success', message: ' Review given successfully.' });
+    } catch (e) {
+        console.log(e);
+
+        return res.json({
+            code: 500,
+            status: 'error',
+            error: e,
+        });
+    }
+};
+export default async function handler(req: NextApiRequest, res: any) {
+    const { method } = req;
```

Comment:
@fazlerabby07 bhai handler.get diye korlei mone hoy aktu clean besi dekhay. switch ta kemon jani hard to read mone hocche

Reply:
@sifatul fix it.